src/piidigger/filehandlers/_sharedfuncs.py

```diff
@@ -2,26 +2,56 @@
 ############ Shared File Handler Functions ###########
 ######################################################
 
+class ContentHandler:
+    def __init__(self, 
+                 maxContentSize: int
+                ):

```

Comment:
```suggestion
    def __init__(self, maxContentSize: int):

```

I'm not sure if this is just a python thing but this feels a bit weird to me being all spaced out.